### PR TITLE
exclude `meetup-m` from using a `--small` variant shape

### PR DESCRIPTION
**Pull request by akdetrick**
The `meetup-m` logo icon uses the same path for all sizes. This branch introduces a whitelist that excludes both `meetup-m` and `external-[service]` icons from small variants.

### CODE REVIEW

**Comment by akdetrick** [comment]
Marked **Prelim** until we merge #219 which I don't want to do right as I sign off for the day.

File: src/media/icon.test.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ describe('Icon', () => {
 		});
 
 		it('renders small shape variant for "xs" icons', () => {
-			const xsIconShape = 'test';
+			const xsIconShape = 'location';
 			const actual = getIconShape(xsIconShape, 'xs');
 			const expected = `${xsIconShape}--small`;
```
Comment (eilinora) [inline src/media/icon.test.jsx]: added a const for `--small`

**Comment by akdetrick** [comment]
Tests are now up to date with master, which uses a `const` for the thin SVG style suffix, `--small`

File: src/media/icon.test.jsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ describe('Icon', () => {
 		});
 
 		it('renders small shape variant for "xs" icons', () => {
-			const xsIconShape = 'test';
+			const xsIconShape = 'location';
```
Comment (eilinora) [inline src/media/icon.test.jsx]: This icon has changed in the latest swarm icons build it's now `location-pin` i believe

Comment (akdetrick) [inline src/media/icon.test.jsx]: Thanks - bumped for passing tests.

**Pull request merged**